test/unit/alphabet/cigar/cigar_op_test.cpp

```diff
@@ -0,0 +1,44 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+#include "../alphabet_test_template.hpp"
+#include "../alphabet_constexpr_test_template.hpp"
+#include <seqan3/alphabet/cigar/cigar_op.hpp>
+
+INSTANTIATE_TYPED_TEST_CASE_P(cigar_op, alphabet, cigar_op);
+INSTANTIATE_TYPED_TEST_CASE_P(cigar_op, alphabet_constexpr, cigar_op);
+
+TEST(cigar_op, to_char_assign_char)
+{
+	EXPECT_EQ(to_char(cigar_op{}.assign_char('M')), 'M');
+	EXPECT_EQ(to_char(cigar_op{}.assign_char('D')), 'D');
+	EXPECT_EQ(to_char(cigar_op{}.assign_char('I')), 'I');
+	EXPECT_EQ(to_char(cigar_op{}.assign_char('S')), 'S');
+	EXPECT_EQ(to_char(cigar_op{}.assign_char('H')), 'H');
+	EXPECT_EQ(to_char(cigar_op{}.assign_char('N')), 'N');
+	EXPECT_EQ(to_char(cigar_op{}.assign_char('P')), 'P');
+	EXPECT_EQ(to_char(cigar_op{}.assign_char('X')), 'X');
+	EXPECT_EQ(to_char(cigar_op{}.assign_char('=')), '=');
```

Comment:
Here you could use a loop over the chars of `"MDISHNPX="`?

Reply:
I mean a two-liner like
```cpp
for (char chr : std::string{"MDISHNPX="})
    EXPECT_EQ(to_char(cigar_op{}.assign_char(chr)), chr);
```

Reply:
Ah I see. Done!